idaes/ui/flowsheet.py

```diff
@@ -233,9 +234,9 @@ def _construct_stream_labels(self):
 
                 for k, v in var_value.items():
                     if k is None:
-                        label += f"{var} {value(v)}\n"
+                        label += f"{var} {round(value(v), 5)}\n"
```

Comment:
Since the rounding is done a few times in the file, you could consider DRYing it in some way, e.g. a module-level constant set to 5, or a utility function `lambda x: round(x, 5)`

Reply:
Minor add: using a constant instead of a function would allow to reuse the value for functions other than the builtin `round()`, e.g. `pd.DataFrame.round()`.

Reply:
I don't think that a lambda function is super necessary here so I just I added a module level constant set to 5.